libethereum/Executive.h

```diff
@@ -78,6 +78,21 @@ class StandardTrace
 	DebugOptions m_options;
 };
 
+
+struct AccountSnapshot
+{
+	bool exists = false;
+	bool isCreation = false;
+	int nonceInc = 0;
+	Address address;
```

Comment:
Actually it is the same purpose -- this is the address of currently processing account (it is newly created address for CREATE, it receives sent ethers, etc). The caller it the message sender.